quiche/quic/core/http/spdy_utils.cc

```diff
@@ -40,7 +40,7 @@ bool SpdyUtils::ExtractContentLengthFromHeaders(int64_t* content_length,
       if (!absl::SimpleAtoi(value, &new_value) ||
           !quiche::QuicheTextUtils::IsAllDigits(value)) {
         QUIC_DLOG(ERROR)
-            << "Content length was either unparseable or negative.";
+            << "Content length was either unparsable or negative.";
```

Comment:
I'm not attached to particular spellings -- both were present, at least some places mark the extra `e` variant as the alternate, e.g. https://wikidiff.com/unparsable/unparseable